src/bat/confirmations/issuer_info.cc

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/confirmations/issuer_info.h"
+
+namespace confirmations {
+
+IssuerInfo::IssuerInfo() :
+    name(""),
```

Comment:
Nitpick: don't really need to do this, as std::string initializes to empty by default.